src/test/java/teammates/test/pageobjects/InstructorHomePage.java

```diff
@@ -36,8 +35,16 @@
     @FindBy(className = "button_sortenddate")
     private List<WebElement> tablesSortByEndDate;
 
+    
+    public InstructorCopyFsToModal fsCopyModal;
+    
     public InstructorHomePage(Browser browser){
         super(browser);
+        if (!browser.driver.findElements(By.id("fsCopyModal")).isEmpty()) {
```

Comment:
This condition seems like it'll fit in `InstructorCopyFsToModal.java` more, to reduce the reliance on that id string.